avidemux_ru.ts

```diff
@@ -5933,482 +5937,482 @@ Do you want to load it  ?
     </message>
     <message>
         <location line="+1"/>
         <source>Temporal AQ</source>
-        <translation type="unfinished"></translation>
+        <translation>Временное адаптивное квантование (AQ)</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Weighted Prediction</source>
-        <translation type="unfinished"></translation>
+        <translation>Взвешенное предсказание</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Even with HEVC encoding support present, lossless presets and B-frames may be unavailable with older hardware</source>
-        <translation type="unfinished"></translation>
+        <translation>Даже при наличии поддержки кодека HEVC предустановки без потерь и B-кадры могут быть недоступны на старом аппаратном обеспечении</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Even with H.264 encoding support present, lossless presets may be unavailable with older hardware</source>
-        <translation type="unfinished"></translation>
+        <translation>Даже при наличии поддержки кодека H.264 предустановки без потерь могут быть недоступны на старом аппаратном обеспечении</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Lookahead and Adaptive Quantization may be unavailable with older hardware</source>
-        <translation type="unfinished"></translation>
+        <translation>Предсказание и адаптивное квантование могут быть недоступны на старом аппаратном обеспечении</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Rate Control</source>
-        <translation type="unfinished"></translation>
+        <translation>Управление битрейтом</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Frame Control</source>
-        <translation type="unfinished"></translation>
+        <translation>Управление кадрами</translation>
     </message>
     <message>
         <location line="+26"/>
         <source>Basic Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Общие настройки</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Advanced Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Расширенные настройки</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>NVENC H.264 configuration</source>
-        <translation type="unfinished"></translation>
+        <translation>Настройки NVENC H.264</translation>
     </message>
     <message>
         <source>nvEnc HEVC  configuration</source>
-        <translation type="obsolete">Настройка nvEnc HEVC</translation>
+        <translation type="obsolete">Настройки nvEnc HEVC</translation>
```

Comment:
`type="obsolete"` means the source doesn't exist anymore. It will be purged together with the translation once lupdate is run with `-no-obsolete` switch. It is probably not worth effort fixing such translations.